### DISCUSSION TITLE
请求将电暖器设备的属性设置为温控器

### DISCUSSION THREAD

**Discussion by s-leeq** [XiaoMi/ha_xiaomi_home#31] [Ideas]
如图，现在的电暖器设备还是分散的参数配置，甚至温度都是number类别，希望能将其和空调一样进行配置。
![IMG_2528](https://github.com/user-attachments/assets/f1f20ba6-a4ee-4536-82f4-66563610a1b2)

**Comment by JimmyKmi** [comment]
将会适配，参考: https://github.com/XiaoMi/ha_xiaomi_home/discussions/18

**Comment by SusanPhevos** [comment]
同 #209

**Comment by s-leeq** [comment]
#209
